bcl2fastq/index_summary.py

```diff
@@ -5,100 +5,133 @@
         ),
     ]),
+    #           Sample_run_hidden holds json split format of "Known"
+    #               columns: "FlowCell","Index1","Index2","LIMS IUS SWID","LaneClusterPF","LaneClusterRaw",
+    #                   "LaneNumber","LaneYield","QualityScoreSum","ReadNumber","Run","RunNumber","SampleID",
+    #                   "SampleName","SampleNumberReads","SampleYield","TrimmedBases","Yield","YieldQ30"
+    #               Index: 6761 - 8307
+
+    #           pruned_unknown_hidden holds json split format of "unknown"
+    #               columns:  "Count","LaneNumber","Index1","Index2","Run","LIMS IUS SWID"
+    #               Index: 218463 - 227804
+
     html.Div(id='sample_run_hidden', style={'display': 'none'}),
     html.Div(id='pruned_unknown_hidden', style={'display': 'none'}),
-            #html.div is just a way of dividing the HTML into meaningful groups
-            #Within the main layout division there are 4 groups: 
-            #           1. the section with the 100% bar graph
-            #           2.  Nested subdivision of: 
-            #                   a) pie graph at 24%
-            #                   b) bar graph at 76%
-            #           3. Sample Run hidden
-            #           4. Pruned  Unknown Hidden
-            #Note: span can be used for inline setting
+
 ])
 
 try:
-    from app import app 
-except ModuleNotFoundError: 
+    from app import app
+except ModuleNotFoundError:
     import dash
-    app = dash.Dash(__name__) 
+
+    app = dash.Dash(__name__)
     app.layout = layout
 
 
 @app.callback(
+
     dep.Output('sample_run_hidden', 'children'),
     [dep.Input('run_select', 'value')]
-#           Whenever input(drop down) changes, app.callback re-runs and updates     
+
 )
-def update_sample_run_hidden(run_alias):        
+#           Replace run variable iteratively
+def update_sample_run_hidden(run_alias):
+    """ When input (run_select) is changed, function is rerun to get json data for selected run
+
+        Attributes:
+            run(list): run number
+
+        Parameters:
+            run_alias(list): alias of the run
+
+        Returns:
+             json object with columns, indices, and data, from run variable
+"""
     run = index[index['Run'] == run_alias]
     run = run[run['ReadNumber'] == 1]
     run = run[~run['SampleID'].isna()]
     run = run.drop_duplicates(['SampleID', 'LaneNumber'])
     return run.to_json(date_format='iso', orient='split')
-#           Replace run variable iteratively 
 
 
 @app.callback(
     dep.Output('pruned_unknown_hidden', 'children'),
     [dep.Input('run_select', 'value')]
 )
 def update_pruned_unknown_hidden(run_alias):
+    """ When input (run_select) is changed, function prune_unknown_index_from_run is run within
+        update_pruned_unknown_hidden to get json data for selected run
+
+        Attributes:
+            pruned(list): select run_alias'
```

Comment:
I'm a bit unclear on what `select run alias'` means, could you clarify or remove it? (Same with method below)